### PR TITLE
Update `web` readme

### PR DESCRIPTION
**Pull request by Etheryte**
## What does this PR change?

Delete the old outdated readme, add a new one.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: This PR adds documentation.  

- [x] **DONE**

## Test coverage
- No tests: Documentation.

- [x] **DONE**

## Links

Part of https://github.com/SUSE/spacewalk/issues/18162

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: web/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+For the legacy web UI, please see the `java` subdirectory.  
+
+The `web` subdirectory consists of roughly the following main chunks:  
+
+ - `web/po`: Translations.
+ - `web/html/javascript`: Legacy scripts, most of them global. Over time we're slowly trying to sunset these piece by piece.  
+ - `web/html/src/branding`: Branding assets, such as stylesheet sources, fonts, images etc.
+ - `web/html/src/build`: Build tooling for the web UI.
+ - `web/html/src/components`, `web/html/src/core`, `web/html/src/manager`, `web/html/src/utils`: Source code for the web UI.
+ - `web/html/dist`: Output directory for the frontend build, do not check this directory in nor modify it directly, your changes will be overwritten by the next build.
```
Comment (juliogonzalez) [inline web/README.md:11]: Should we mention that `web/modules` is obsolete, scheduled for removal, and must not be used?

Comment (juliogonzalez) [reply web/README.md:11]: Check with other developers, but I am 99.99% sure it's not needed.

Comment (Etheryte) [reply web/README.md:11]: I'm not familar with those parts, hopefully someone else can chime in on that. If it's scheduled for removal, is there a ticket for it?

Comment (juliogonzalez) [reply web/README.md:11]: I think there was a card a couple of years ago about a cleanup for stuff from spacewalk, but probably it's closed alreeady and we'd need a new one.

For now, at least we should document.

@mcalmer am I right about `web/modules`?

**Review by cbbayburt — APPROVED** [review]

**Review by ncounter — APPROVED** [review]

**Pull request merged**